templates/destination-router.cfg.xml.erb

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<destination-router xmlns='http://openrepose.org/repose/destination-router/v1.0'>
```

Comment:
This needs fixin'.  In 7 it's been converted to docs.openrepose.org but you'll need to keep 6 backwards compatibility.  You may want to look at how this is handled for other files.